packages/iTwinUI-react/src/core/Buttons/IconButton/IconButton.tsx

```diff
@@ -46,15 +46,12 @@ export const IconButton: IconButtonComponent = React.forwardRef(
     return (
       <Element
         ref={ref}
-        className={cx(
-          'iui-button',
-          `iui-${styleType}`,
-          {
-            [`iui-${size}`]: !!size,
-            'iui-active': isActive,
-          },
-          className,
-        )}
+        className={cx('iui-button', className)}
+        data-iui-variant={
+          styleType && styleType !== 'default' ? styleType : undefined
+        }
+        data-iui-size={size}
+        data-iui-active={isActive}
```

Comment:
hmm... Interesting.. in comment, we said `isActive default value is false`, but we do not assign it in props destruction...
But it makes sense not to have attribute with `false` value, so lets leave it, I guess...